bookstore.py

```diff
@@ -192,11 +192,13 @@ def get_book_by_id(self, id):
             
             if book_data:
                 book = Book(book_data['title'], book_data['author'], book_data['read'], book_data['rowid'])
-                    
-            con.close()            
-            
-            return book 
+            else:
+                book = None # Wyatt: If the book_data isn't found in the database, book is set to None
```

Comment:
You don't need your name here. Git can report who wrote any line of code in the project.